js/app.js

```diff
@@ -0,0 +1,28 @@
+$(function() { 
+  
+    $("#stopButton").click(turnBulbRed);
```

Comment:
@AmyMirisola this selector could have been `$('.bulb).css("background-color", "black"); since all the light have this class